addons/portal/models/res_lang.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, models
+from odoo.http import request
+
+
+class Lang(models.Model):
+    _inherit = "res.lang"
+
+    @api.model
+    def get_available(self):
+        langs = super().get_available()
+        if not request or not request.is_frontend:
+            return langs
+        # Only return the active ones in this case
+        return [lang for lang in langs if lang[3]]
```

Comment:
@JKE-be if you can have a look at the first commit of this PR again, I don't know what I am doing :) I still don't understand that orm_cache thing in this case: portal overrides the method and returns a filtered result depending if `is_frontend` or not... so what is really cached? or it there a way to indicate that it should be cached depending on request.is_frontend?